app/views/profiles/_form.html.erb

```diff
@@ -18,6 +18,39 @@
                    label: "Nickname",
                    help: "Unique. Reserved for future use on menadevs.com" %>
 
+
+
+<label>Skills</label>
+<select id="skill_list" name="skill_list[]" class="skills-input" class="js-example-basic-single" multiple="multiple">
+
+  User.where(weekly_subscriber: true).find_each do |user|
+    NewsMailer.weekly(user).deliver_now
+  end
+
+
+  <% Tag.where(is_approved: true).find_each do |skill| %>
+    <option value="<%= skill.name %>"><%= skill.name %></option>
+  <% end %>
+</select>
+<span class="help-block skills-help">Note: if a skill doesn't exist in the list, manually type it and press enter.</span>
+
+<script type="text/javascript">
```

Comment:
JS tags are included at the end of the view if they're needed to be included once in a very specific way.
I suggest we move this one to a file under `app/assets/javascripts/` and add it to the list of JS files that are loaded under `assets.rb`